treeherder/push_health/utils.py

```diff
@@ -55,6 +55,19 @@ def clean_test(test_name):
     return clean_name
 
 
+def clean_config(config):
+    if config == 'pgo':
```

Comment:
add a comment here that we find that pgo ~= opt and this helps us get a more representative sample size of data.

Reply:
I think you could also put this and `clean_platform` in a serializer too :)

Reply:
I'm curious about the use of serializers here.  These transformations are done well before output from the API.  They're done to help the "history" and "current" data to match up (as well as for better display once it gets out of the API.  

I'll admit that I haven never written a Serializer, but aren't they usually for transforming data right as you're returning the JSON?  I could use some tutoring here, if it sounds like they're the right approach.  :)  Would this mean I'd need to move most of the code from the ``/push_health/push_health.py`` file into the view?  And then I would do my comparison and extra processing afterward?

Reply:
Yes, serializers are used to convert the data to json but you can do a lot of transformations there (which may not be relevant for everything you're doing in that loop). Here's how I've used `to representation`: https://github.com/mozilla/treeherder/blob/master/treeherder/webapp/api/serializers.py#L272

That's used by the `FailuresByBugSerializer` (the class below - line 285 in that file)

You can also use the `SerializerMethodField` which is used by `name` in line 203 here: https://github.com/mozilla/treeherder/blob/master/treeherder/webapp/api/performance_serializers.py#L214

DRF has a pretty useful guide, but there's a lot to look through: https://www.django-rest-framework.org/api-guide/serializers/

Once you create a serializer, you need to override the serializer class on your viewset to use the one you created.

Reply:
>  Would this mean I'd need to move most of the code from the `/push_health/push_health.py` file into the view? And then I would do my comparison and extra processing afterward?

I don't think you'd have to move it all into the view, but you could move the processing and transformations into the Serializer.